flax/linen/fp8_ops.py

```diff
@@ -117,9 +120,19 @@ def __repr__(self) -> str:
 fm32 = fp8_meta_dtype_wrapper(jnp.float32)
 
 def get_fp8_max(fp8_dtype, out_dtype):
-  assert fp8_dtype in (jnp.float8_e4m3fn, jnp.float8_e5m2)
+  assert fp8_dtype in (jnp.float8_e4m3fn, jnp.float8_e5m2,
+                       jnp.float8_e4m3fnuz, jnp.float8_e5m2fnuz)
   return jnp.finfo(fp8_dtype).max.astype(out_dtype)
 
+def get_fp8_dtypes(fp8_genre):
```

Comment:
this function seems to only be used by tests, can we just move it into the test file?

Reply:
Done.